openvdb/openvdb/tools/Filter.h

```diff
@@ -5,8 +5,7 @@
 ///
 /// @file tools/Filter.h
 ///
-/// @brief Filtering of VDB volumes. Note that only the values in the
-/// grid are changed, not its topology! All operations can optionally
+/// @brief Filtering of VDB volumes. All operations can optionally
 /// be masked with another grid that acts as an alpha-mask.
```

Comment:
This should comment in the new version that topology *can* be modified if setProcessTiles is set (and comment that it is not set by default, if I read it right?)

Likewise, mention the dilation is "chunky" (if it is...)

Reply:
> Likewise, mention the dilation is "chunky" (if it is...)

Updated the comment, though I'm not sure what you're referring to with this!